docs/usage.md

```diff
@@ -185,6 +185,69 @@ Options:
           Print help (see a summary with '-h')
 ```
 
+## registry diff
+
+```
+Generate a diff between two versions of a semantic convention registry.
+
+This diff can then be rendered in multiple formats:
+- a console-friendly format (default: ansi),
+- a structured document in JSON or YAML format,
+- ...
+
+Usage: weaver registry diff [OPTIONS] --baseline-registry <BASELINE_REGISTRY> [OUTPUT]
```

Comment:
could I use it with weaver yaml and pipe output to jq/jinja similarly to the registry? 

It would allow generating area-specific migration guides and doing any further customizations in the same way as we do codegen.

(either way, it's not  a blocker)

Reply:
Yes, it's already in place :-)

You can see several examples in the `defaults/diff_templates`.